jujugui/static/gui/src/app/app.js

```diff
@@ -747,6 +747,23 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Renders the SearchResults component to the page in the designated element.
+
+      @method _renderSearchResults
+      @param {String} query The search query.
+    */
+    _renderSearchResults: function(query) {
+      var visible = query ? true : false;
+      React.render(
+        <window.juju.components.WhiteBox
```

Comment:
In my branch I created a global `var components = window.juju.components;` once you rebase this to trunk you can update to that, it looks a little cleaner :)